doc/usecases/polyculture/search_plants.md

```diff
@@ -0,0 +1,36 @@
+# Use Case: Search Plants
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can search for plants to add to her map and sort results by different criterion.
+- **Status:** In Progress
+- **Assignee:** Moritz, Paul
+
+## Scenarios
+
+- **Precondition:**
+  - The user is logged in to the app.
+  - The plants layer or seed entry dialog is shown.
+- **Main success scenario:**
+  - The user types something into the search text box.  
+    This will search for partial matches of:
+    - names of specific plants (specie or variety)
+    - Binomial name (Latin name)
+    - German and English common names
+    - German and English synonyms
+    - Furthermore, other columns can be matched with extra syntax
+  - The results are ranked by:
+    - Environmental fit
+    - Ecological value
+- **Alternative scenario:**
```

Comment:
There are no alternative scenarios, maybe you can incorporate my suggestion: 

- The user refines his search query by adding or removing search terms or using advanced search filters (alphabetical order or plant size).

Reply:
I extended the ordering.
https://github.com/ElektraInitiative/PermaplanT/pull/169/commits/358023b0f04cf9da4074288261da5c35c112c344